platform/ios/CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ Mapbox welcomes participation and contributions from everyone. Please read [CONT
 * Added an `MGLSymbolStyleLayer.symbolZOrder` property for forcing point features in a symbol layer to be layered in the same order that they are specified in the layer’s associated source. ([#12783](https://github.com/mapbox/mapbox-gl-native/pull/12783))
 * Fixed a crash when a style layer `*-pattern` property evaluates to nil for a particular feature. ([#12896](https://github.com/mapbox/mapbox-gl-native/pull/12896))
 * Fixed an issue with view annotations (including the user location annotation) and the GL map lagging relative to each other. ([#12895](https://github.com/mapbox/mapbox-gl-native/pull/12895))
+* Fixed an issue where fill and line layers would occasionally flicker on zoom due to a bug in `ClipIDGenerator` ([#12982](https://github.com/mapbox/mapbox-gl-native/pull/12982))
```

Comment:
Small nit: since these changelogs are meant to be end-developer facing, it’s usually not necessary to mention internal implementation details. In this case, I’d suggest lopping off the “due to a bug in `ClipIDGenerator`”, as that’s only relevant to us.